plugins/device/rest.go

```diff
@@ -151,6 +151,12 @@ func deviceEditHandler(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	device := r.URL.Query().Get("device")
```

Comment:
Query parameters are mostly used for optional parameters. Please add device to URL itself or request body.

Reply:
Use as part of url itself